api-report/container-runtime.api.md

```diff
@@ -145,6 +145,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     get IFluidTokenProvider(): IFluidTokenProvider | undefined;
     get isDirty(): boolean;
     static load(context: IContainerContext, registryEntries: NamedFluidDataStoreRegistryEntries, requestHandler?: (request: IRequest, runtime: IContainerRuntime) => Promise<IResponse>, runtimeOptions?: IContainerRuntimeOptions, containerScope?: FluidObject, existing?: boolean): Promise<ContainerRuntime>;
+    static load2(context: IContainerContext, registryEntries: NamedFluidDataStoreRegistryEntries, existing: boolean, requestHandler?: (request: IRequest, runtime: IContainerRuntime) => Promise<IResponse>, runtimeOptions?: IContainerRuntimeOptions, containerScope?: FluidObject): Promise<ContainerRuntime>;
```

Comment:
two thoughts:

1. If we are breaking this, we should move to the object patterns `load2(params:{ context, registryEntries,...})` such that future breaking changes are easier. I'm not worried about object allocation perf here as we the number of calls here is small; once per container
2. the existing flag already exist on the IContainerContext, so why do we need to explictly provide it at all.

Reply:
2 was answered by looking at the bug. the PR description should also include that